src/PKSim.Core/Services/ParticleApplicationCreator.cs

```diff
@@ -77,6 +75,12 @@ private void addDrugStartFormulaForParticles(IApplicationBuilder applicationBuil
             setupParticleBin(applicationBuilder, i, binRadius[i - 1], binNumberOfParticlesFactor[i - 1], formulaCache);
          }
 
+         //// add insoluble molecule for particle bin
```

Comment:
?? Why is this still there?

Reply:
Once we have adjusted our application/formulation structure, we can think about reducing the number of state variables by moving Insoluble drug fro Bin-Level to Application (or better: formulation) level.
Then those lines would be used.